.github/workflows/ci-prb.yml

```diff
@@ -1,7 +1,7 @@
 name: PR Builder
 on:
   pull_request:
-    branches: [ main ]
+    branches: [ main, '0.41' ]
```

Comment:
Because it's easy, decided to enable all actions for 0.41, just in case we will need to send a PR for that branch only.